apps/website/src/pages/docs/header.mdx

```diff
@@ -14,68 +14,50 @@ import * as AllExamples from 'examples';
+The first thing most people look at when they arrive at a new web page is the header. It is usually stuck at the top of the page. The header contains vital information about your company's name and function, as well as the purpose of the specific page.
 
-Header with circular callouts, numbered from 1 to 8.
+## Usage
 
-- Services menu
-- Home page
-- Enterprise portal
-- Assets menu
-- Projects menu
-- Notifications
-- Help
-- User icon
+### `<Header>` component
 
-### Actions
+It consists of various subcomponents and can be customized by modifying its props.
 
-- Services menu pops up a list of services available to the user.
-- Home page takes you to CONNECT Center.
-- Enterprise Portal takes you to the Subscription Services Portal.
-- Assets menu takes you to the assets page.
-- Projects menu takes you to the ProjectWise Projects page.
-- Notifications pops up your list of notifications.
-- Help takes you to Bentley Communities for Bentley Cloud and Web Services.
-- User profile is a drop down with options for your user profile.
+- Logo can be added using the `appLogo` prop and specifying the desired logo under the `<HeaderLogo>` subcomponent.
+- Breadcrumbs trail can be implemented using the `breadcrumbs` prop and `<HeaderBreadcrumbs>` subcomponent. Individual buttons can be added within the breadcrumbs trail using the `<HeaderButton>` subcomponent. For a dropdown menu, the `menuItems` prop can be used inside the `<HeaderButton>` for showing items and `onClick()` property for handling actions.
+- Buttons/Icons/Content can be displayed towards the left of menuItems using the `actions` prop.
+- `menuItems` are typically an array of items used to create dropdown menus, allowing users to select from a list of options or perform specific actions. When passed as a prop, it expects a list of MenuItems components. Each `MenuItem` has a `value` (the label displayed) and an `onClick` function that takes the `close` argument (for closing).
 
-## Secondary header
+### `<HeaderButton>` subcomponent
 
-The secondary header contains the product icon, product name, iModel name and the name of the initially opened view. On the right side of the bar is the settings menu.
+It is designed to offer a range of behaviors that are dependent on the props that are passed to it.
 
-Secondary header with circular callouts, numbered from 1 to 5.
+- `onClick` prop: behaves like a standard button.
+- When set as='a' and `href` prop: behaves like a regular anchor link.
+- `menuItems` prop: behaves like a dropdown button.
+- Both `menuItems` and `onClick/href` prop: behaves like a split button, where the left part is the main button/link and the right part opens a dropdown menu.
+- `isActive` prop: behaves as an active button.
 
-- Product icon
-- Product name
-- iModel name
-- View name
-- Settings
-
-### Actions
+<LiveExample src='Header.full.tsx'>
```

Comment:
Since we are mentioning that `<HeaderButton>` can be made a link, would be good to show it in the example too.